src/components/login/AuthDetails.tsx

```diff
@@ -0,0 +1,28 @@
+import { onAuthStateChanged, signOut } from "firebase/auth";
```

Comment:
Import User from "firebase/auth" so that you can use it with your authUser hook.
Like this:
`import { onAuthStateChanged, signOut, User } from 'firebase/auth';`

Reply:
done